packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -126,25 +131,35 @@ export class SOQLEditorInstance {
     webviewPanel.onDidDispose(this.dispose, this, this.subscriptions);
   }
 
+  protected sendMessageToUi(
+    type: string,
+    payload?: string | string[] | DescribeSObjectResult
+  ): void {
+    this.webviewPanel.webview
+      .postMessage({
+        type,
+        payload
+      })
+      .then(undefined, (err: string) => {
```

Comment:
This rejected block has never thrown.  It's theoretical.  I even tried messing with CSP.  Still couldn't get it to hit.  But this would be catastrophic to our app if we can't communicate across the webview iframe....so i left it.  The error would look like SIMILAR to this in the channel and instruct the user to open an issue.

<img width="1003" alt="Screen Shot 2020-12-08 at 10 37 08 AM" src="https://user-images.githubusercontent.com/599418/101825394-c82bbd00-3aea-11eb-8a0b-ac64d3f7ace8.png">